packages/app-page-builder/src/editor/plugins/elementSettings/grid/index.tsx

```diff
@@ -15,7 +16,7 @@ export default [
         name: "pb-editor-page-element-style-settings-grid-settings",
         type: "pb-editor-page-element-style-settings",
         render() {
-            return <GridSettings />;
+            return isLegacyRenderingEngine ? <></> : <GridSettings />;
```

Comment:
It can be done...
But column wrap or wrap direction are NOT a part of legacy engine for a VERSION 5.33.5 from the doc. 
https://github.com/webiny/webiny-js/blob/dev/docs/LEGACY_PB_DEVELOPMENT.md

Column wrap or wrap direction were initially done on old engine and migrated to a new engine latter (but I am not sure if old engine was released with them or not)

Also I am trying to simplify this task and PR (as it is already huge, and testing is not simple to check all grid cases)

CC: @SvenAlHamad maybe you have a suggestion about this?

Reply:
No worries, we're aware this PR blew in scope and complexity (hence the thorough testing and code review from our side).

The column wrap and wrap direction actually WERE also introduced with the legacy rendering engine. That's how, to my surprise, I actually discovered that I needed to have this same feature implemented with the new one. 🙂 

So, I was adding it based on what I saw in the old rendering engine. 

Ultimately, because of this fact, some Grid settings should still be rendering when a user is using the legacy rendering engine. In other words, `GridSettings` component should always be rendered. It's just that some of the code within it should not be rendered with the new engine (the stuff you added in this PR).